drivers/usb/device/usb_dc_stm32.c

```diff
@@ -293,11 +295,16 @@ static int usb_dc_stm32_clock_enable(void)
 #if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_usbphyc)
 	LL_AHB1_GRP1_EnableClock(LL_AHB1_GRP1_PERIPH_OTGHSULPI);
 	LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_OTGPHYC);
+#else
+#ifdef CONFIG_SOC_SERIES_STM32H7X
+	__HAL_RCC_USB2_OTG_FS_ULPI_CLK_DISABLE();
+	__HAL_RCC_USB2_OTG_FS_ULPI_CLK_SLEEP_DISABLE();
```

Comment:
Please check if still required after fixing the compatible to "otgfs"

Reply:
This was no longer necessary with `otgfs`, removed.